Dockerfile

```diff
@@ -8,6 +8,12 @@ ARG UNSTRUCTURED
 RUN yum -y update && \
   yum -y install poppler-utils xz-devel which
 
+# Enable the EPEL repository
+RUN yum install -y epel-release && yum clean all
+
```

Comment:
guessing centos dependency fun here? curios: was this needed for issues fetching pandoc? or something else?

Reply:
Yes this is a pandoc necessity, but the tests using pandoc are still complaining. Going to keep investigating!

Reply:
Argh! 😞 I definitely feel your pain there!